vendor/brave-ios/Ledger/Generated/NativeLedgerClient.h

```diff
@@ -35,7 +35,9 @@ class NativeLedgerClient : public ledger::LedgerClient {
   void LoadState(const std::string & name, ledger::OnLoadCallback callback) override;
   void LoadURL(const std::string & url, const std::vector<std::string> & headers, const std::string & content, const std::string & contentType, const ledger::URL_METHOD method, ledger::LoadURLCallback callback) override;
   std::unique_ptr<ledger::LogStream> Log(const char * file, int line, const ledger::LogLevel log_level) const override;
-  void OnGrantFinish(ledger::Result result, ledger::GrantPtr grant) override;
+  void OnExcludedSitesChanged(const std::string & publisher_id, ledger::PUBLISHER_EXCLUDE exclude) override;
```

Comment:
As mentioned in Slack: These 3 adds to all 3 native* files seem to be accidentally added back during a rebase